app/src/main/java/com/eveningoutpost/dexdrip/utils/DexCollectionType.java

```diff
@@ -130,6 +132,8 @@ public static boolean hasBattery() {
         return usesBattery.contains(getDexCollectionType());
     }
 
+    public static boolean hasSensorStartStopSupport() { return supportsSensorStartStop.contains(getDexCollectionType()); }
```

Comment:
Again: please use consistent code formatting! The function should look like the other functions:

```java
public static boolean hasSensorStartStopSupport() {
    return supportsSensorStartStop.contains(getDexCollectionType());
}
```